plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp

```diff
@@ -115,6 +117,13 @@ void USentrySubsystem::Initialize()
 
 	PromoteTags();
 	ConfigureBreadcrumbs();
+
+	OutputDevice = MakeShareable(new FSentryOutputDevice());
+	if(OutputDevice)
+	{
+		GLog->AddOutputDevice(OutputDevice.Get());
+		GLog->SerializeBacklog(OutputDevice.Get());
```

Comment:
Is there a chance for this to get overwritten by the user or something else?

Reply:
I don't think so - having multiple output devices is totally acceptable as they don't interfere with each other.